src/test/scala/mesosphere/marathon/instance/InstanceIdTest.scala

```diff
@@ -34,27 +38,36 @@ class InstanceIdTest extends UnitTest {
 
     "InstanceIds can be converted from TaskIds with container name" in {
       val appId = "/test/foo/bla/rest".toPath
-      val parsedTaskId = Task.Id("test_foo_bla_rest.instance-myinstance.someContainerName")
+      val uuid = UUID.fromString("b6ff5fa5-7714-11e7-a55c-5ecf1c4671f6")
+      val parsedTaskId = Task.Id.fromIdString("test_foo_bla_rest.instance-b6ff5fa5-7714-11e7-a55c-5ecf1c4671f6.someContainerName")
```

Comment:
This is an important change. Is `test_foo_bla_rest.instance-myinstance.someContainerName` a valid id? If so the uuid would be `myinstance`.